### DISCUSSION TITLE
0.5.0 Install Problems

### DISCUSSION THREAD

**Discussion by Stoneman101** [flybywiresim/aircraft#2538] [Help] [answered]
Question, I always thought that when I was using the installer I was doing right.  I guess not.  My MSFS  is on it's own drive and I see the community folder F:MSFS2020/community. That is where all addons  and liveries are.  When I install A32NX with the installer, I'm not seeing the changes in the cockpit, nor am I seeing the FBW liveries.  Ideas?

**Comment by Benjozork** [comment]
Can you show a screenshot of your Community folder ?

**Reply by Stoneman101** [reply]
![Capture](https://user-images.githubusercontent.com/76258832/102697004-7b3d9a00-4200-11eb-9c36-93d1adf4306b.PNG)

**Reply by Stoneman101** [reply]
![Capture](https://user-images.githubusercontent.com/76258832/102697041-9c9e8600-4200-11eb-91a1-96aca5ccd144.PNG)

**Comment by PortPilot** [comment]
I do have the same problem after the latest MS update and the latest master files. Tried to delete all and again, but no success.

The liveries are under \Community\A32NX\SimObjects\AirPlanes**, but but are also not visible in FS. But more interesting to have is the missing cockpit :)

Any ideas?

Best regards
Jan

![community](https://user-images.githubusercontent.com/52661081/103156990-e6faa680-47ae-11eb-8e34-c2de57cad5f5.JPG)

**Comment by PortPilot** [comment, marked as answer]
The installer will not install if the community folder is at a different drive. So I downloaded everything on the desktop and copied it to the community folder.

And suprise, it works!

regards
Jan
